src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -164,7 +172,7 @@ public async Task<IActionResult> List(ListContentsViewModel model, PagerParamete
                 {
                     if (ctd.GetSettings<ContentTypeSettings>().Listable)
                     {
-                        var authorized = await _authorizationService.AuthorizeAsync(User, Permissions.EditContent, await _contentManager.NewAsync(ctd.Name));
+                    var authorized = await _authorizationService.AuthorizeAsync(User, CommonPermissions.EditContent, await _contentManager.NewAsync(ctd.Name));
```

Comment:
and are you using `await _contentManager.NewAsync()` or are you using `new ContentItem()`?

Reply:
Doesn't make any difference with the changes I made in the ContentTypeAuthorizationHandler.